lib/model/QubitDigitalObject.php

```diff
@@ -1661,18 +1661,6 @@ public function isImage()
         return self::isImageFile($this->getName());
     }
 
-    /**
-     * Return true if this is a compound digital object.
-     *
-     * @return bool
-     */
-    public function isCompoundObject()
```

Comment:
Is this function no longer being used anywhere?

Reply:
I grepped and could find any use of it, oddly! If you could double check, just in case I missed something, that'd be awesome.